tests/tests_test.go

```diff
@@ -0,0 +1,267 @@
+package tests
+
+import (
+	"context"
+	migapi "github.com/konveyor/mig-controller/pkg/apis/migration/v1alpha1"
+	"github.com/konveyor/mig-controller/pkg/pods"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"log"
+	"path"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"strings"
+	"time"
+)
+
+var ctx = context.Background()
+
+var _ = Describe("Running migmigration when storage, cluster and plan are ready", func() {
+	var (
+		migPlan     *migapi.MigPlan
+		migPlanName string
+		namespaces  []string
+	)
+
+	JustBeforeEach(func() {
+		Expect(hostClient.Create(ctx, migPlan)).Should(Succeed())
+		Eventually(func() string {
+			log.Println("waiting for migPlan to be ready")
+			hostClient.Get(ctx, client.ObjectKey{Name: migPlanName, Namespace: MigrationNamespace}, migPlan)
+			l := len(migPlan.Status.Conditions.List)
+			if l > 0 {
+				return migPlan.Status.Conditions.List[l-1].Type
+			}
+			return ""
+		}, time.Minute*5, time.Second).Should(Equal("Ready"))
+	})
+
+	AfterEach(func() {
+		err = hostClient.Delete(ctx, &migapi.MigPlan{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      migPlanName,
+				Namespace: MigrationNamespace,
+			},
+		})
+		if err != nil {
+			log.Println(err)
+		}
+	})
+
+	Context("Testing BZ #1965421, PVC name is more than 63 char long", func() {
+
+		BeforeEach(func() {
+			migPlanName = "foo"
```

Comment:
I think you declared a constant for this above, but then aren't using it here.

Reply:
`migPlanName` should be different to make sure we don't come across an error of creating `migPlan` with the same name again. That is why I am not using a constant there.